### ISSUE TITLE
docs: Update `RunAxe` `node.Target` example snippet

### ISSUE THREAD

**Issue by Zidious** [dequelabs/axe-core-nuget#143]
Within our [RunAxe docs example](https://github.com/dequelabs/axe-core-nuget/tree/develop/packages/playwright#runaxe) we say that is possible:

```cs
 foreach(var target in node.Target)
{
    Console.WriteLine($"\t\t{target}.");
}
```

Unfortunately this throws the following error: 

> foreach statement cannot operate on variables of type 'AxeSelector' because 'AxeSelector' does not contain a public instance or extension definition for 'GetEnumerator'

We should amend our docs to remove looping of `Node.Target` and simply print it or provide a test case of using our suggested way of using `Node.Target` which is with `ToString`. 

This was reported by Fom in Slack (Good catch!): https://axecommunity.slack.com/archives/C019K5W9S68/p1703073223049639

**Comment by padmavemulapati** [comment]
Verified with the latest build - `Deque.AxeCore.Playwright_Version="4.8.2-alpha.70 `

Correct Code Example:

```
[Test]
        public async Task RunAxe_ForViolationsWithNode()
        {
           
           await Page!.GotoAsync("https://qateam.dequecloud.com/attest/api/test.html");
           AxeResult axeResults = await Page!.RunAxe();
            Console.WriteLine($"Axe ran against {axeResults.Url} on {axeResults.Timestamp}.");

            Console.WriteLine($"Rules that failed:");
                foreach(var violation in axeResults.Violations)
                {
                    Console.WriteLine($"Rule Id: {violation.Id} Impact: {violation.Impact} HelpUrl: {violation.HelpUrl}.");

                    foreach(var node in violation.Nodes)
                    {
                        Console.WriteLine($"\tViolation found at: {node.Target}");
                        Console.WriteLine($"\t...with HTML: {node.Html}");
                    }
                }       
            // Serialize Axe results to JSON
            string jsonResults = JsonConvert.SerializeObject(axeResults, Formatting.Indented);
            
            // Write JSON to a file
            await File.WriteAllTextAsync("axe_results_Node-violations.json", jsonResults);
            
        }
```

We can see the results:

```
Axe ran against https://qateam.dequecloud.com/attest/api/test.html on 3/8/2024 10:43:12 AM +00:00.
Rules that failed:
Rule Id: aria-allowed-attr Impact: critical HelpUrl: https://dequeuniversity.com/rules/axe/4.8/aria-allowed-attr?application=axe-core-playwright-nuget.
        Violation found at: #wcag2a-fail > a[href="#"][role="button"]
        ...with HTML: <a href="#" role="button" aria-checked="true">aria-checked_notallowed4_button</a>
Rule Id: autocomplete-valid Impact: serious HelpUrl: https://dequeuniversity.com/rules/axe/4.8/autocomplete-valid?application=axe-core-playwright-nuget.
        Violation found at: #lname
        ...with HTML: <input id="lname" type="text" autocomplete="my given-name" ...="">
Rule Id: color-contrast Impact: serious HelpUrl: https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=axe-core-playwright-nuget.
        Violation found at: #wcag2aa-fail > p
        ...with HTML: <p style="color:red">This text has BAD contrast with its background</p>
Rule Id: dlitem Impact: serious HelpUrl: https://dequeuniversity.com/rules/axe/4.8/dlitem?application=axe-core-playwright-nuget.
        Violation found at: ["#testFrame", "dt:nth-child(1)"]
        ...with HTML: <dt>Coffee</dt>
        Violation found at: ["#testFrame", "dd:nth-child(2)"]
        ...with HTML: <dd>Black hot drink</dd>
        Violation found at: ["#testFrame", "dt:nth-child(3)"]
        ...with HTML: <dt>Milk</dt>
        Violation found at: ["#testFrame", "dd:nth-child(4)"]
        ...with HTML: <dd>White cold drink</dd>
Rule Id: heading-order Impact: moderate HelpUrl: https://dequeuniversity.com/rules/axe/4.8/heading-order?application=axe-core-playwright-nuget.
        Violation found at: h6
        ...with HTML: <h6>Oops! I am level 6 heading</h6>
Rule Id: image-alt Impact: critical HelpUrl: https://dequeuniversity.com/rules/axe/4.8/image-alt?application=axe-core-playwright-nuget.
        Violation found at: ["#testFrame", "#nestedTestFrame", "img[src$=\"myImage.png\"]"]
        ...with HTML: <img src="myImage.png" width="100" height="100">
        Violation found at: ["#testFrame", "#nestedTestFrame", "img[src$=\"myImage1.png\"]"]
        ...with HTML: <img src="myImage1.png" width="100" height="100">
Rule Id: label Impact: critical HelpUrl: https://dequeuniversity.com/rules/axe/4.8/label?application=axe-core-playwright-nuget.
        Violation found at: input[name="lastname"]
        ...with HTML: <input type="text" name="lastname">
Rule Id: link-in-text-block Impact: serious HelpUrl: https://dequeuniversity.com/rules/axe/4.8/link-in-text-block?application=axe-core-playwright-nuget.
        Violation found at: #experimental-fail-1 > p > a[href="mars2.html?a=last_will"]
        ...with HTML: <a href="mars2.html?a=last_will" style="color: #001fff; text-decoration: none; cursor: pointer;">
            marshmallow-1 is a confectionery</a>
        Violation found at: #experimental-fail-2 > p > a[href="mars2.html?a=last_will"]
        ...with HTML: <a href="mars2.html?a=last_will" style="color: #001fff; text-decoration: none; cursor: pointer;">
            marshmallow-2 is also a confectionery</a>


Passed!  - Failed:     0, Passed:     1, Skipped:     0, Total:     1, Duration: 16 s - tests.dll (net5.0)
```


https://github.com/dequelabs/axe-core-nuget/assets/47409579/08651286-3d4b-46fa-95e3-7b916292e495


## Environment:
| **Label**                         | **Value** |
| -------------------------- | ------ |
| **Product**                | _axe-core-nuget_ |
|**Deque.AxeCore.Playwright** | _" Version="4.8.2-alpha.70_ |
| **nunit-Version**           | _3.13.2_ |
| **Selenium-webdriver-Version**           | _4.15.0_ |
| **Webdriver-Manager-Version**           | _2.17.0_ |
| **OS-Details**      | _MAC- Intel Core i7 - 11.6.8 && Windows 11 _ |
|**BrowserDetails**|_|**BrowserDetails**|_Chrome Version 122.0.6261.112  && Firefox 123.0.1_|_|
